sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -202,6 +203,18 @@ export interface KeyProperties {
 // @public
 export type KeyType = "EC" | "EC-HSM" | "RSA" | "RSA-HSM" | "oct";
 
+// @public
+export class KeyVaultIdentifier implements ParsedKeyVaultIdentifier {
+    constructor(collection: KeyVaultIdentifierCollections, identifier: string);
+    id: string;
+    name: string;
+    vaultUri: string;
+    version?: string;
```

Comment:
We plan to have this class across all the three packages ? 

If yes, then would KeyVaultKeyIdentifier, KeyVaultSecretIdentiifer and KeyVaultCertificateIdentifier, work better ?

@heaths

Reply:
This class is agnostic of the client though, much like our generated code 🤔 I could omit the collections parameter and thus have a KeyVaultKeyIdentifier, and the subsequent classes, instead of this! Let me know if that's better.

Reply:
I prefer a generic one that just had a "Collection" property or something. Fewer types with which to interact.

Reply:
I believe I have addressed your feedback now! Let me know if I've missed or misinterpreted something.

Reply:
Even though I will have to copy this code to each client, I think I have addressed Heath's feedback at this point. Please review again!